src/invidious/views/template.ecr

```diff
@@ -153,6 +153,11 @@
                         <span><%= translate(locale, "Current version: ") %> <%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %> @ <%= CURRENT_BRANCH %></span>
                     </div>
                 </div>
+            <% if CONFIG.footer %>
+            <div>
+               <h4 style="text-align: center;"><%= CONFIG.footer %></h3>
```

Comment:
Any reason you chose an h4 over a span element? Also, the end tag doesnt match the start tag (h4/h3)

Reply:
I also tried with a span element. It doesn't look right imo:
<img width="1083" alt="image" src="https://github.com/iv-org/invidious/assets/125497673/205c28aa-118b-4791-aaa3-4785a39fba18">
I would think that it would fit better with the banner message and the rest of the page.
I also tried both h3 and h4 and forgot to change it on the other side (oops).